setup.py

```diff
@@ -28,16 +28,17 @@
     long_description=readme,
     classifiers=[
         'Programming Language :: Python',
-        'Programming Language :: Python :: 3.5',
         'Programming Language :: Python :: 3.6',
         'Programming Language :: Python :: 3.7',
+        'Programming Language :: Python :: 3.8',
         'Framework :: AsyncIO',
     ],
     packages=find_packages(),
     install_requires=None,
     extras_require={
         'redis:python_version<"3.7"': ['aioredis>=0.3.3'],
-        'redis:python_version>="3.7"': ['aioredis>=1.0.0'],
+        'redis:python_version>="3.8"': ['aioredis>=1.3.0'],
+        'redis:python_version>="3.7" and python_version<"3.8"': ['aioredis>=1.0.0'],
```

Comment:
tests for python3.8 only pass starting aioredis>=1.3.0 so restricting this in the setup file